core/src/main/java/org/apache/accumulo/core/client/AccumuloClient.java

```diff
@@ -363,14 +364,24 @@ AccumuloClient changeUser(String principal, AuthenticationToken token)
    * Builder method for setting Accumulo instance and zookeepers
    */
   interface InstanceArgs {
-    AuthenticationArgs forInstance(String instanceName, String zookeepers);
+    AuthenticationArgs to(CharSequence instanceName, CharSequence zookeepers);
   }
 
   /**
    * Builder methods for creating AccumuloClient using properties
    */
```

Comment:
I think all of these inner interfaces need a since tag

Reply:
Fixed in 23da39e93c3209